src/main/java/com/exasol/sql/expression/rendering/ValueExpressionRenderer.java

```diff
@@ -165,4 +166,16 @@ public void visit(final BooleanExpression booleanExpression) {
         booleanExpression.accept(expressionRenderer);
         append(expressionRenderer.render());
     }
+
+    @Override
+    public void visit(final ExasolCastFunction castFunction) {
+        appendKeyword("CAST");
+        startParenthesis();
+        castFunction.getValue().accept(this);
+        appendKeyword(" AS");
+        final ColumnsDefinitionRenderer columnsDefinitionRenderer = new ColumnsDefinitionRenderer(this.config);
+        castFunction.getType().accept(columnsDefinitionRenderer);
+        append(columnsDefinitionRenderer.render());
```

Comment:
Extract to a method, we use the same logic in this class already, remove the duplication

Reply:
Not possible since two different interfaces for accept: `DataType#accept() `and `ColumnsDefinition#accept`

Reply:
At least extract the method that creates a ColumnsDefinitionRenderer

Reply:
:+1: